### PR TITLE
M5StickC support added

### PR DESCRIPTION
**Pull request by toto99303**
M5StickC support added

### CODE REVIEW

**Comment by gyengus** [comment]
Thank you so much
IMG_20231217_190409

**Review by gyengus — APPROVED** [review]
It works

**Comment by toto99303** [comment]
Awesome! Thanks, too! It was fun!

File: src/media/myFonts.h
Comment (BitMaker-hub) [inline src/media/myFonts.h:1]: We should take care of amount flash used on old boards like standard ESP32.
Try to avoid filling shared libs

**Review by BitMaker-hub — APPROVED** [review]

**Pull request merged**